.vscode/extensions.json

```diff
@@ -0,0 +1,12 @@
+{
+  "recommendations": [
+    "charliermarsh.ruff",
+    "magicstack.magicpython",
+    "ms-python.debugpy",
+    "ms-python.mypy-type-checker",
+    "ms-python.python",
+    "ms-python.vscode-pylance",
+    "njpwerner.autodocstring",
+    "tamasfe.even-better-toml"
```

Comment:
We can also add `cucumberopen.cucumber-official` for feature file syntax highlighting